package.json

```diff
@@ -16,17 +16,18 @@
   },
   "license": "ISC",
   "devDependencies": {
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1",
```

Comment:
not that I rly know what `react-dom` is and how it was used before, but how come both `react` and `react-dom` are being removed here?

Reply:
The list of dependencies was just reordered. Nothing was removed. I think it was when I ran npm install --save-dev it sorted the dependcies.

Reply:
okidoki